### PR TITLE
[jaeger-operator] Adds ability to just deploy Jaeger if another operator is available already 

### PR DESCRIPTION
**Pull request by koalalorenzo**
#### What this PR does
It will allow us to deploy Jaeger when an operator is already deployed.

#### Checklist

- [x] [DCO](https://github.com/jaegertracing/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Commits are [GPG signed](https://docs.github.com/en/github/authenticating-to-github/about-commit-signature-verification)
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (`[jaeger]` or `[jaeger-operator]`)

### CODE REVIEW

**Comment by koalalorenzo** [comment]
@naseemkullah Please let me know if it makes sense and if I am doing things correctly to contribute to the repo! 
BTW, Thank you so much for this helm chart!!! :)

**Comment by naseemkullah** [comment]
seems great @koalalorenzo, thanks for contributing, but i only help maintain the vanilla jaeger chart, the operator chart is maintained by @cpanato and @batazor

**Comment by koalalorenzo** [comment]
@cpanato @batazor, do you have any input? I have another (private) PR waiting for this one to be merged. If I can't have this merged in the main chart, I will have to rely on a fork, and I would love to avoid that 😅 I don't want to push too much but, is it possible to get it merged?

**Comment by cpanato** [comment]
@koalalorenzo Sorry for the delay to replay :(. i will review this today

File: charts/jaeger-operator/templates/deployment.yaml
Comment (cpanato) [inline charts/jaeger-operator/templates/deployment.yaml:73]: nit: new line

**Review by cpanato — CHANGES_REQUESTED** [review]

**Comment by cpanato** [comment]
I did not fully understand the use case for this, why you will install this helm which is to install the operator and you don't install it.

**Comment by koalalorenzo** [comment]
Oh wow! Thank you a lot! :)

The biggest help is using helm to roll back changes. The use case is to have Helm manage both the operator and the instance or in different releases (so that the operator has its own release and the various instances their own). This helps as other helm charts that we are using (ex: Prometheus operator) already support this feature and help roll back changes via helm.
Does it make sense? 😅

File: charts/jaeger-operator/templates/deployment.yaml
Comment (koalalorenzo) [reply charts/jaeger-operator/templates/deployment.yaml]: Fixed!

File: charts/jaeger-operator/Chart.yaml
Comment (cpanato) [inline charts/jaeger-operator/Chart.yaml:4]: why you bump to 2.18.2 and not just 2.18.0?

Comment (koalalorenzo) [reply charts/jaeger-operator/Chart.yaml:4]: I had some issues on my own fork when running GitHub Actions 🤦 I am sorry for that! Should I switch to 2.18.0?

Comment (cpanato) [reply charts/jaeger-operator/Chart.yaml:4]: yes please

Comment (koalalorenzo) [reply charts/jaeger-operator/Chart.yaml]: No problem! 😄 Done!

**Review by cpanato — APPROVED** [review]
i dont have strong opinions on this, this lgtm

**Comment by koalalorenzo** [comment]
Do you know how this can be merged?

**Comment by cpanato** [comment]
After @batazor review :)

**Comment by koalalorenzo** [comment]
Any update @batazor ? :)

**Comment by batazor** [comment]
@koalalorenzo Honestly, I don't see how this change will help us.
can you explain in a little more detail?

**Comment by koalalorenzo** [comment]
Here: https://github.com/jaegertracing/helm-charts/pull/175#issuecomment-737137467

Honestly, it took **too much to have some feedback or review** that we decided to not use the operator at all, for the time being.

**Pull request closed without merging**
